packages/ui-date-time-input/src/DateTimeInput/README.md

```diff
@@ -0,0 +1,148 @@
+---
+describes: DateTimeInput
+---
+
+A DateTimeInput component is used to enter a date-time value. It is built as a composition
+of the [DateInput](#DateInput) and [TimeSelect](#TimeSelect) components.
+
+A DateTimeInput with `column` layout and a default value:
+
+```js
+---
+example: true
+render: false
+---
+class Example extends React.Component {
```

Comment:
I found another keyboard nav issue: if I tab to the date select, then hit up or down arrow, it opens the dropdown, but doesn't highlight the correct day. Also, when I tab out, it doesn't save the date on blur, just resets to the former value.